mongodbatlas/provider.go

```diff
@@ -218,3 +221,16 @@ func expandLabelSliceFromSetSchema(d *schema.ResourceData) []matlas.Label {
 	}
 	return res
 }
+
+func getPluginVersion() string {
```

Comment:
I think there is a better way to get the Plugin Version, I'll take a look at other providers to find it. In the meantime LGTM!

Reply:
Thank you so much! 👍